lib/main.dart

```diff
@@ -0,0 +1,140 @@
+import "package:flutter_dotenv/flutter_dotenv.dart";
+import "package:openapi_generator_annotations/openapi_generator_annotations.dart";
+import "package:oss_surveys_customer/api/api_factory.dart";
+import "package:oss_surveys_customer/mqtt/mqtt_client.dart";
+import "package:simple_logger/simple_logger.dart";
+
+final logger = SimpleLogger();
+final apiFactory = ApiFactory();
+
+void main() async {
+  _configureLogger();
+  await dotenv.load(fileName: ".env");
+  mqttClient.connect();
+  runApp(const MyApp());
+}
+
+/// Configures logger to use [logLevel] and formats log messages to be cleaner than by default.
+void _configureLogger({logLevel = Level.INFO}) {
+  SimpleLogger().setLevel(logLevel);
+  SimpleLogger().formatter = ((info) => "[${info.time}] -- ${info.callerFrame ?? "NO CALLER INFO"} - ${info.message}");
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: "Flutter Demo",
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // Try running your application with "flutter run". You"ll see the
+        // application has a blue toolbar. Then, without quitting the app, try
+        // changing the primarySwatch below to Colors.green and then invoke
+        // "hot reload" (press "r" in the console where you ran "flutter run",
+        // or simply save your changes to "hot reload" in a Flutter IDE).
+        // Notice that the counter didn"t reset back to zero; the application
+        // is not restarted.
+        primarySwatch: Colors.blue,
+      ),
+      home: const MyHomePage(title: "Flutter Demo Home Page"),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key, required this.title});
+
+  // This widget is the home page of your application. It is stateful, meaning
+  // that it has a State object (defined below) that contains fields that affect
+  // how it looks.
+
+  // This class is the configuration for the state. It holds the values (in this
+  // case the title) provided by the parent (in this case the App widget) and
+  // used by the build method of the State. Fields in a Widget subclass are
+  // always marked "final".
+
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _counter = 0;
+
+  void _incrementCounter() {
+    setState(() {
+      // This call to setState tells the Flutter framework that something has
+      // changed in this State, which causes it to rerun the build method below
+      // so that the display can reflect the updated values. If we changed
+      // _counter without calling setState(), then the build method would not be
+      // called again, and so nothing would appear to happen.
+      _counter++;
+    });
+  }
```

Comment:
Maybe there's no need to keep this default code increment feature?

Reply:
Probably. I did leave them to be able to see reference later on how the UI's are built, but maybe it can be checked elsewhere when the time is right. Removed and resolved.